dcos/constants.go

```diff
@@ -20,7 +20,7 @@ const (
 func GetFileDetectIPLocation() string {
 	switch runtime.GOOS {
 	case "windows":
-		return "/opt/mesosphere/bin/detect_ip.ps1"
+		return "c:\\dcos\\opt\\bin\\detect_ip.ps1"
```

Comment:
Please add a unit test for this.

Reply:
@makkes, the reason for this change is a request of @janisz on PR [https://github.com/dcos/dcos-diagnostics/pull/164] to update dcos-go with accordance with mentioned PR. Oleksandr Bielov isn't a GO developer, not sure that he should/can be responsible for the unit test

Reply:
Writing unit tests is essential part of writing the code.

Reply:
I agree with @janisz